packages/next-swc/crates/next-dev-tests/tests/integration/next/postcss/WEB-1016/input/postcss.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
```

Comment:
There is an existing test cases packages/next-swc/crates/next-core/src/next_client/context.rs

Reply:
Are we running this test case with Turbopack today?